### PR TITLE
Allow setting Subject session information (closes #78)

### LINKED ISSUE #78: Add ability to set Domain Session ID/Index
**Is your feature request related to a problem? Please describe.**
It's currently possible to set the Domain User ID on a `Subject`, but not the other session information that is usually stored in the same cookie.

**Describe the solution you'd like**
It should be possible to set the Domain Session ID and Domain Session Index fields on the Subject.

**Describe alternatives you've considered**
Explicitly creating a `client_session` entity to include the same information.

**Additional context**
N/A

### PR DESCRIPTION
**Pull request by jethron**

### CODE REVIEW

File: tracker/subject.go [hunk 1]
```diff
@@ -77,7 +80,34 @@ func (s Subject) SetDomainUserId(domainUserId string) {
...
+func (s Subject) SetDomainSessionIndex(domainSessionIndex int) {
+	s.payload.Add(DOMAIN_SIDX, common.NewString(common.IntToString(domainSessionIndex)))
+}
+
 // SetNetworkUserId adds a network user id to the key-value store.
 func (s Subject) SetNetworkUserId(networkUserId string) {
 	s.payload.Add(NETWORK_UID, common.NewString(networkUserId))
 }
+
+// FromIdCookie sets subject fields from the ID cookie to the key-value store.
```
Comment (matus-tomlein) [inline tracker/subject.go:98]: Nice touch!

**Review by matus-tomlein — APPROVED** [review]
LGTM!

**Comment by matus-tomlein** [comment]
Hi @jethron, sorry forgot about this one, but happy to make the release if you merge in the PR into the release branch.

**Comment by jethron** [comment]
Thanks @matus-tomlein; I don't actually have permissions on this repo to do the merge, are you able to?

**Pull request merged**